games/devtest/mods/unittests/password_hash.lua

```diff
@@ -0,0 +1,25 @@
+local function test_password_hash()
```

Comment:
personally I would have put this in `misc.lua`

Reply:
There doesn't seem to be much rhyme or reason on what unit tests get their own files and what goes in the one big file. Is this a change request or just a comment?

Reply:
maybe should refactor the unit tests so each one is in file with a name that matches the lua_api or cpp_api implementation file name.

Reply:
I made the change anyways